### ISSUE TITLE
Local Install Issues

### ISSUE THREAD

**Issue by daviddarnes** [Cinematico/cinematico#15]
I just tried installing locally. The data was written into the relevant files, but I'm met with a 'Not Found' error when I'm sent to the /settings page.

I'm not sure whether these notices are a factor but they might be related:

```
Notice: Undefined variable: network_install in /Users/daviddarnes/Sites/cinematico/index.php on line 20
Notice: Undefined index: page in /Users/daviddarnes/Sites/cinematico/cinematico/includes/functions.php on line 294
Notice: Undefined index: page in /Users/daviddarnes/Sites/cinematico/cinematico/includes/functions.php on line 294
Notice: Undefined index: page in /Users/daviddarnes/Sites/cinematico/cinematico/includes/functions.php on line 294
Notice: Undefined variable: site_image in /Users/daviddarnes/Sites/cinematico/index.php on line 133
Notice: Undefined index: filename in /Users/daviddarnes/Sites/cinematico/index.php on line 168
Notice: Undefined index: filename in /Users/daviddarnes/Sites/cinematico/index.php on line 173
Notice: Undefined index: page in /Users/daviddarnes/Sites/cinematico/cinematico/includes/functions.php on line 294
Notice: Undefined index: page in /Users/daviddarnes/Sites/cinematico/cinematico/includes/functions.php on line 294
Notice: Undefined index: page in /Users/daviddarnes/Sites/cinematico/cinematico/includes/functions.php on line 294
```

**Comment by jptksc** [comment]
This has all been fixed in 1.0.4.

**Comment by daviddarnes** [comment]
Awesome, thanks @Circa75!
